dstep/Configuration.d

```diff
@@ -77,6 +77,10 @@ struct Configuration
     /// do not translate following symbols
     @("skip", "Skip translation of <symbol>.")
     string[] skipSymbols;
+
+    /// enable printing of warnings
+    @("enable-warnings", "Enable printing of warnings [default].")
+    bool enableWarnings = true;
```

Comment:
True, but I was thinking that it could be used for cases where DStep currently outputs `<unimplemented>` as well. But perhaps that should be separate flags.